test/tests/functional/pbs_soft_walltime.py

```diff
@@ -945,16 +951,9 @@ def test_soft_less_cput(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
         # verify that job is deleted when cput limit is reached
-        self.logger.info("Sleep for 10 waiting for cput to cause job to be deleted")
```

Comment:
Why the removal of the log message?  It's kind to your test runner to tell them why they are sleeping.

Reply:
It was my debug statement.

Reply:
Could you put it back?  I don't view these as debug statements.  I view them as telling your test runner what is going on.  You  could even merge it into the comment right above it since they both really serve the same purpose.

Reply:
Sure!  I can add the log message back. Can you clarify what you mean by "merge it into the comment right above it since they both really serve the same purpose"?  I don't understand.

Reply:
You have a code comment to tell the person reading the code what you are sleeping for, and then you have a log message which is telling the test runner what you are sleeping for.  If you express them both in the log message, you can remove the code comment.